mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -1705,6 +1741,102 @@ resource "google_compute_instance_group_manager" "igm-basic" {
 `, network, template, target, igm)
 }
 
+<% unless version == "ga" -%>
+func testAccInstanceGroupManager_stoppedSuspendedTargetSize(network, template, igm string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_network" "sr-igm" {
+  name = "%s"
+}
+
+resource "google_compute_instance_template" "sr-igm" {
+  name           = "%s"
+  machine_type   = "e2-medium"
+  disk {
+    source_image = data.google_compute_image.my_image.self_link
+    auto_delete  = true
+    boot         = true
+    device_name  = "stateful-disk"
+  }
+  network_interface {
+    network = "default"
+  }
+}
+
+resource "google_compute_instance_group_manager" "sr-igm" {
+  description = "Terraform test instance group manager"
+  name        = "%s"
+
+  version {
+    instance_template = google_compute_instance_template.sr-igm.self_link
+    name              = "primary"
+  }
+
+  base_instance_name        = "tf-test-sr-igm"
+  zone                      = "us-central1-c"
+  target_size               = 2
+  standby_policy {
+    initial_delay_sec           = 20
+    mode                        = "MANUAL"
+  }
+  target_suspended_size         = 2
+  target_stopped_size           = 1
+}
+`, network, template, igm)
+}
+
+func testAccInstanceGroupManager_stoppedSuspendedTargetSizeUpdate(network, template, igm string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_network" "sr-igm" {
+  name = "%s"
+}
+
+resource "google_compute_instance_template" "sr-igm" {
+  name           = "%s"
+  machine_type   = "e2-medium"
+  disk {
+    source_image = data.google_compute_image.my_image.self_link
+    auto_delete  = true
+    boot         = true
+    device_name  = "stateful-disk"
+  }
+  network_interface {
+    network = "default"
+  }
+}
+
+resource "google_compute_instance_group_manager" "sr-igm" {
+  description = "Terraform test instance group manager"
+  name        = "%s"
+
+  version {
+    instance_template = google_compute_instance_template.sr-igm.self_link
+    name              = "primary"
+  }
+
+  base_instance_name        = "tf-test-sr-igm"
+  zone                      = "us-central1-c"
+  target_size               = 2
+  standby_policy {
+    initial_delay_sec           = 30
```

Comment:
here you can test changing initial_delay to 0

Reply:
Removed the fields so that the default value (0) should pass.